Gruntfile.js

```diff
@@ -133,17 +133,17 @@ module.exports = function(grunt) {
       },
       /* Changes in theme dir rebuild sphinx */
       sphinx: {
-        files: ['sphinx_rtd_theme/**/*', 'demo_docs/**/*.rst', 'demo_docs/**/*.py'],
+        files: ['sphinx_rtd_theme/**/*', 'docs/**/*.rst', 'docs/**/*.py'],
```

Comment:
Since README is now also built, `README.rst` should be added to wacthed files.

Reply:
Can we just check the full docs file? `docs/**/*` wont this catch everything?

Reply:
README.rst is not in the `docs` dir, it's in the root.